f3dex2.s

```diff
@@ -884,164 +965,180 @@ G_VTX_handler:
     lhu     outputVtxPos, (vertexTable)(outputVtxPos)
     move    inputVtxPos, $20
     lbu     $8, mvpValid
-    andi    $6, $5, G_LIGHTING_H
-    bnez    $6, Overlay23LoadAddress    // This will always end up in overlay 2, as the start of overlay 3 loads and enters overlay 2
+    andi    tmpCurLight, $5, G_LIGHTING_H  // If no lighting, tmpCurLight is 0, skips transforming light dirs and setting this up as a pointer
+    bnez    tmpCurLight, Overlay23LoadAddress  // This will always end up in overlay 2, as the start of overlay 3 loads and enters overlay 2
      andi   $7, $5, G_FOG_H
-f3dzex_000017BC:
-    bnez    $8, g_vtx_load_mvp          // Skip recalculating the mvp matrix if it's already up-to-date
-     sll    $7, $7, 3                   // $7 is 8 if G_FOG is set, 0 otherwise
-    sb      cmd_w0, mvpValid
-    li      $21, pMatrix
+After_LightDirMVTransposeNormalize:
```

Comment:
I'd prefer if label names were snake case; the command handler labels like `G_VTX_handler` are only named like that so that they have the same capitalization as the command they handle (G_VTX in that case).

Reply:
There are other label names besides the handlers which use different casing (e.g. `Overlay3Address`), but no problem, I've converted all the labels I added/changed to snake case.